htdocs/fichinter/card.php

```diff
@@ -605,7 +605,8 @@
 		// Extrafields
 		$extrafields->fetch_name_optionals_label($object->table_element_line);
 		$array_options = $extrafields->getOptionalsFromPost($object->table_element_line);
-		$objectline->array_options = $array_options;
+		$objectline->fetch_optionals();
```

Comment:
The fetch_optionals() has already be done into the 
if ($objectline->fetch($lineid)

Reply:
You're right. Because every fetch() should contains the fetch_optionals(), I pushed a different fix that include the fetch_optional inside the fetch (like for other objects).